setup.py

```diff
@@ -21,6 +21,7 @@
         'License :: OSI Approved :: Apache Software License',
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```

Comment:
Then I guess we should update Travis to run also on 3.5 and be more specific. Right?